packages/near-fast-auth-signer/src/components/AuthCallback/AuthCallback.tsx

```diff
@@ -230,10 +229,14 @@ function AuthCallbackPage() {
           setStatusMessage(isRecovery ? 'Recovering account...' : 'Creating account...');
 
           // claim the oidc token
-          window.fastAuthController = new FastAuthController({
-            accountId,
-            networkId
-          });
+          if (window.fastAuthController) {
```

Comment:
This was to prevent from re-declaring the `FastAuthController` class. If we re-declare, existing in memory key pair gets flushed out. Instead, just update account id if logic reaches upto this point.

Reply:
Got it, can we create something re-usable for this? This code it's being repeated 3 times on your PR

Reply:
updated the syntax